app/presenters/two_factor_options_presenter.rb

```diff
@@ -26,6 +26,15 @@ def options
       backup_code_option + webauthn_option + piv_cac_option
   end
 
+  def selected_options
+    [TwoFactorAuthentication::WebauthnPlatformSelectionPresenter.new(user: user)] +
+      [TwoFactorAuthentication::SetUpAuthAppSelectionPresenter.new(user: user)] +
+      [TwoFactorAuthentication::PhoneSelectionPresenter.new(user: user)] +
+      [TwoFactorAuthentication::BackupCodeSelectionPresenter.new(user: user)] +
+      [TwoFactorAuthentication::WebauthnSelectionPresenter.new(user: user)] +
+      [TwoFactorAuthentication::PivCacSelectionPresenter.new(user: user)]
```

Comment:
Rather than concatenating a bunch of arrays, could we just have a single array with all of the entries?

```suggestion
    [
      TwoFactorAuthentication::WebauthnPlatformSelectionPresenter.new(user: user)],
      TwoFactorAuthentication::SetUpAuthAppSelectionPresenter.new(user: user)],
      TwoFactorAuthentication::PhoneSelectionPresenter.new(user: user)],
      TwoFactorAuthentication::BackupCodeSelectionPresenter.new(user: user)],
      TwoFactorAuthentication::WebauthnSelectionPresenter.new(user: user)],
      TwoFactorAuthentication::PivCacSelectionPresenter.new(user: user),
    ]
```

Reply:
I like that much better.